spec/controllers/course/material/folders_controller_spec.rb

```diff
@@ -30,5 +31,20 @@
         end
       end
     end
+
+    describe '#update_files' do
+      subject { patch :update_files, course_id: course, id: folder_stub, material_folder: { files:[]} }
```

Comment:
Line is too long. [103/100]<br>Space missing after colon.<br>Space inside } missing.